code/datums/jobs/job/marines.dm

```diff
@@ -73,6 +94,31 @@ Your squaddies will look to you when it comes to construction in the field of ba
 	id = /obj/item/card/id/dogtag
 	back = /obj/item/storage/backpack/marine/satchel
 
+/datum/outfit/job/marine/engineer/equipped
+	name = "Squad Engineer (Equipped)"
+
+	belt = /obj/item/storage/belt/utility/full
+	ears = /obj/item/radio/headset/mainship/marine
+	w_uniform = /obj/item/clothing/under/marine/engineer
+	wear_suit = /obj/item/clothing/suit/storage/marine
+	shoes = /obj/item/clothing/shoes/marine
+	gloves =/obj/item/clothing/gloves/marine
```

Comment:
recommend you replace the gloves line with this: gloves = /obj/item/clothing/gloves/marine/insulated